packages/core/src/fhir-deduplication/shared.ts

```diff
@@ -234,3 +234,13 @@ export function pickMostDescriptiveStatus<T extends string>(
   }
   return status;
 }
+
+export function isKnownCoding(coding: Coding) {
+  const code = coding.code?.trim().toLowerCase();
+  const system = coding.system?.trim().toLowerCase();
+  const display = coding.display?.trim().toLowerCase();
+  if (code === "unk" || system?.includes("unknown") || display === "unknown") {
```

Comment:
system?.includes("unknown") &&
    code?.includes(unknownCoding.code.toLowerCase()) &&
    display === unknownCoding.display.toLowerCase() &&
    (!text || text === unknownCode.text.toLowerCase())
    
    
    shouldnt we do this here too?